test/scripts/esports_data_from_rimble.txtar

```diff
@@ -5,7 +5,12 @@ cp stdout result-match-winner.json
 
 # [check] assert stdout contains correct match information
 exec jq -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result-match-winner.json
-cmp stdout expected-output-match-winner.json
+cp stdout output.json
+exec jq -e '.Success == true' output.json
+exec jq -e '.Error == ""' output.json
+exec jq -e '.Value.MatchID' output.json
+exec jq -e '.Value.Date' output.json
+exec jq -e '.Value.Winner' output.json
```

Comment:
Because the match data is subject to change I can only check for the expected field names not their actual values. I tried doing some fancy regex stuff I could not get commands with multi-line strings to work with testscript/txtar